.github/workflows/release.yml

```
@@ -0,0 +1,161 @@
+          release_name: Release ${{ steps.calver.outputs.version }}
+          body_path: release_notes.md
+          draft: false
+          prerelease: false
+          
+  notify-discord-release:
+    needs: [create-release]
+    name: Notify Discord (new release)
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)